src/probnum/randprocs/covfuncs/_covariance_function.py

```diff
@@ -438,6 +451,9 @@ def linop(
             Can also be set to :data:`None`, in which case the function will behave as
             if ``x1 == x0`` (potentially using a more efficient implementation for this
             particular case).
+        use_keops
+            Optional keyword flag to choose whether to use a KeOps-based implementation
+            or the standard implementation.
```

Comment:
Personally, I would remove this argument and just stick to the default behavior outlined above.
If we really need to force a standard implementation at some point, then a setting in `probnum.config` allows for more fine-grained control.
But I can't really think of a situation in which one would want to use a NumPy-based implementation if KeOps is available to be honest.
Thoughts @timweiland @JonathanWenger?

Reply:
Agreed. One could check the `matrix_free` config option here. But currently the default is set to `False` which is not the behavior we would want here.

Reply:
> But I can't really think of a situation in which one would want to use a NumPy-based implementation if KeOps is available to be honest.

I agree :+1:
 Just wasn't sure about it, and it made testing a little easier.